### DISCUSSION TITLE
Support for PostgreSQL 16 planned?

### DISCUSSION THREAD

**Discussion by johanneskastl** [zalando/postgres-operator#2498] [General]
Hi all,

I just found out about this nice operator over the weekend and successfully switched over to it on some of my clusters.

I cannot convert one database, as it is already on PostgreSQL 16. I have searched the issues and discussions but found no trace, if support for this is already planned, already in the works or whatever the state is.

Can someone point me to such a discussion/issue, if one exists? Or is this tracked somewhere else, e.g. SPILO or similar?

Thanks in advance,
Johannes

**Comment by jankatins** [comment]
FYI: PG 16 support is in since https://github.com/zalando/postgres-operator/releases/tag/v1.11.0 released in march 2024
